### PR TITLE
shades can no longer cast or be used in runes

### PR DESCRIPTION
**Pull request by ynot01**
# General Documentation

### Intent of your Pull Request

Remove the rune-slave shade meta

### Why is this change good for the game?

Being kept in someone's pocket and used as a rune-slave to cast multiple-cultist runes is neither balanced nor engaging.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Shades can no longer cast or be used in runes

### What general grouping does this PR fall under? 
Antagonists, Blood Cult

# Changelog

:cl:  
tweak: Shades can no longer cast or be used in runes
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
whats the points of shades now wandering ghosts that move super slow and cant do jack shit and can be oneshot very fun experience

File: code/modules/antagonists/cult/runes.dm [hunk 1]
```diff
@@ -132,6 +135,8 @@ structure_check() searches for nearby cultist structures required for the invoca
 						continue
 				if(L.stat)
 					continue
+				if(!istype(user, /mob/living/simple_animal/shade))
```
Comment (SomeguyManperson) [inline code/modules/antagonists/cult/runes.dm]:
```suggestion
				if(istype(user, /mob/living/simple_animal/shade))
```
continue skips the rest of the loop for that iteration and is used to make things NOT get counted as invokers here

**Comment by ChesterTheCheesy** [comment]
> 
> 
> whats the points of shades now wandering ghosts that move super slow and cant do jack shit and can be oneshot very fun experience

how about putting them into a shell and make into a construct

**Comment by JamieD1** [comment]
@SomeguyManperson

**Comment by boodaliboo** [comment]
> > whats the points of shades now wandering ghosts that move super slow and cant do jack shit and can be oneshot very fun experience
> 
> how about putting them into a shell and make into a construct

then whats the point of shades at all? why would we have them if they are going to be completely and utterly useless there basically a role to revive a human for like 15 seconds intill they get hit once by a floor tile

**Comment by alexkar598** [comment]
Does this affect manifest ghost?

**Comment by ynot01** [comment]
> Does this affect manifest ghost?

no

> > > whats the points of shades now wandering ghosts that move super slow and cant do jack shit and can be oneshot very fun experience
> > 
> > 
> > how about putting them into a shell and make into a construct
> 
> then whats the point of shades at all? why would we have them if they are going to be completely and utterly useless there basically a role to revive a human for like 15 seconds intill they get hit once by a floor tile

shades are meme ghosts that were never intended to have actual use, make them into a construct

**Review by jcatjmeow — APPROVED** [review]
Looks fine, makes sense

**Review by JamieD1 — APPROVED** [review]
